libs/gi-localization/assets/locales/en/char_Furina.json

```
@@ -0,0 +1,12 @@
+{
+  "fanfarePoints": "Fanfare",
+  "memberMult_": "Salon Members Mult.",
+  "numOffer": "Number of characters who offered HP",
+  "c1Inc": "Max Fanfare Increase",
+  "c1Bonus": "Fanfare on Cast",
+  "a4MemberDmg": "Salon Member DMG Bonus",
+  "a4HealInterval": "Singer of Many Waters Healing Interval",
+  "c2Points": "Fanfare points Above the limit",
```

```suggestion
  "c2Points": "Fanfare above the limit",
```
Inconsistent naming and capitalization